provider/source/src/units/info.rs

```diff
@@ -60,15 +59,10 @@ impl DataProvider<UnitsInfoV1> for SourceDataProvider {
             conversion_info_map.insert(unit_name.as_bytes().to_vec(), convert_unit_index);
         }
 
-        let units_conversion_trie =
-            ZeroTrieSimpleAscii::try_from(&conversion_info_map).map_err(|e| {
-                DataError::custom("Could not create ZeroTrie from units.json data")
-                    .with_display_context(&e)
-            })?;
+        // TODO: now we need to create `MeasureUnitParser` in order to create the conversion info. how to do that ?
```

Comment:
@robertbastian , @Manishearth 
Now, in order to create the ConversionInfoV1, we need to have a measure unit parser, and measure unit parser is depending on `UnitsInfoV1`, how can I do that ?

Reply:
I don't understand the question: things can depend on each other just fine

Reply:
done